app/Console/Commands/SetCollectionToLegacyCallNumberSequence.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Jitterbug\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use Jitterbug\Models\NewCallNumberSequence;
+
+class SetCollectionToLegacyCallNumberSequence extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'reset_sequence:legacy {archival_identifier}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'This script deletes all new_call_number_sequences of the specified archival identifier.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+      $archivalIdentifier = $this->argument('archival_identifier');
+
+      // Find all new call number sequences that have been used and get their associated prefix
+      $usedSequencePrefixes = DB::table('new_call_number_sequences')
+        ->where('archival_identifier', $archivalIdentifier)
+        ->where('next', '>', 1)
+        ->get()
```

Comment:
Minor q: Do you need get(), or can you go right to pluck()?

Reply:
i just tested this and i don't need the `get()` and i had no idea! changing it now.